skins/neowx-material/forecast.inc

```diff
@@ -76,6 +76,22 @@
                           $x[1].temperature.min.format("%.0f") / $x[1].temperature.max.format("%.0f")
                         </div>
                       #end if
+                      #if ($variable == "sun") or ($variable == "uv-sun")
+                        <div class="col-2 p-0">
+                          <i class="wi wi-day-sunny $Extras.color-text" title="$pgettext("Forecast","uv") / $pgettext("Forecast","sunshine_duration") / $pgettext("Forecast","daylight_duration")" data-toggle="tooltip" data-html="true"></i>
+                        </div>
+                        <div class="col-10 text-left">
+                          #set daylight_hours = int($x[1].sun.daylight_duration.raw / 3600)
+                          #set daylight_minutes = int(($x[1].sun.daylight_duration.raw % 3600) / 60)
+                          #set sunshine_hours = int($x[1].sun.sunshine_duration.raw / 3600)
+                          #set sunshine_minutes = int(($x[1].sun.sunshine_duration.raw % 3600) / 60)
+                          #if ($variable == "sun")
+                            ${sunshine_hours}:${sunshine_minutes}h / ${daylight_hours}:${daylight_minutes}h
```

Comment:
One more think. Doesn't this work too:
$x[i].sun.sunshine_duration.format("%H:%M")
Since we use ValueHelper it should be possible to use all weewx formating and unit conversion and agregation features